tests/phpunit/integration/tests/REST_API/Stories_Media_Controller.php

```diff
@@ -95,6 +98,8 @@ public function tearDown() {
 
 		$this->remove_caps_from_roles();
 
+		$this->set_permalink_structure( '' );
```

Comment:
This is not needed here. Our base `TestCaase` class does this already.

Reply:
Are you sure, I copied this another REST API endpoint. 
https://github.com/google/web-stories-wp/blob/232a82d63fc9b2068244bc223235fbe3bc282bdc/tests/phpunit/integration/tests/REST_API/Stories_Users_Controller.php#L63-L86

Reply:
I am sure, yes. Check the `TestCase` class and its `tearDown()` method.

Sounds like you can remove this from the `Stories_Users_Controller` tests too.

Reply:
Fixed.